modules/hivemq-edge-module-file/src/main/java/com/hivemq/edge/adapters/file/FileProtocolAdapterFactory.java

```diff
@@ -30,14 +38,51 @@ public class FileProtocolAdapterFactory implements ProtocolAdapterFactory<FileAd
     }
 
     @Override
-    public @NotNull ProtocolAdapter createAdapter(final @NotNull ProtocolAdapterInformation adapterInformation, @NotNull final ProtocolAdapterInput<FileAdapterConfig> input) {
+    public @NotNull ProtocolAdapter createAdapter(
+            final @NotNull ProtocolAdapterInformation adapterInformation,
+            @NotNull final ProtocolAdapterInput<FileAdapterConfig> input) {
         return new FilePollingProtocolAdapter(adapterInformation, input);
     }
 
+    @Override
+    public @NotNull FileAdapterConfig convertConfigObject(
+            final @NotNull ObjectMapper objectMapper, final @NotNull Map<String, Object> config) {
+        if (config.get("FileToMqtt") != null || config.get("mqttToFile") != null) {
```

Comment:
this wont work ;) must be:
```suggestion
        if (config.get("fileToMqtt") != null || config.get("mqttToFile") != null) {
```